isthmus/src/test/java/io/substrait/isthmus/Substrait2SqlTest.java

```diff
@@ -58,7 +146,10 @@ private void test(String query) throws Exception {
     Rel pojoRel2 =
         SubstraitRelVisitor.convert(RelRoot.of(relnodeRoot, SqlKind.SELECT), EXTENSION_COLLECTION);
 
-    Assertions.assertEquals(pojoRel, pojoRel2);
+    if (assertPojo) {
```

Comment:
If never using, remove this condition and argument. Best if we don't need it (though we might eventually)